README.md

```diff
@@ -3,26 +3,39 @@
 [![Codacy Badge](https://api.codacy.com/project/badge/Grade/87238907485845eb879bd61c65561775)](https://www.codacy.com/app/sangaman/xud?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=ExchangeUnion/xud&amp;utm_campaign=Badge_Grade)
+[![License: AGPL v3](https://img.shields.io/badge/License-AGPL%20v3-blue.svg)](https://www.gnu.org/licenses/agpl-3.0)
 
-The Exchange Union Daemon (`xud`) powers [Exchange Union](https://www.exchangeunion.com/), a decentralized exchange (DEX) built on the [Lightning](http://lightning.network/) and [Raiden](https://raiden.network/) networks to enable instant and trustless cryptocurrency swaps. The vision is to bring individuals and existing, centralized exchanges onto the same network and form one, global liquidity pool. This gives users a choice to either trade directly on the DEX, managing private keys and software stack, *or* to conveniently trade via a trusted exchange. Exchanges participating in the network aggregate the network's liquidity and can provide deeper order books and new trading pairs to their users. `xud` encompasses the following components:
+
+![xud logo](logo.png)
+
+The Exchange Union Daemon ([`xud`](https://github.com/ExchangeUnion/xud)) is the open-source node software powering [Exchange Union](https://www.exchangeunion.com/), a decentralized exchange (DEX) built on the [Lightning](https://lightning.network/) and [Raiden](https://raiden.network/) network.
+
+`xud` brings individuals and exchanges onto the same network to form a single global liquidity pool. This gives users a choice to either trade directly on the DEX by running `xud`, managing private keys and software stack, *or* to conveniently trade via a trusted exchange. Exchanges benefit from access to the network's aggregated liquidity and can provide deeper order books and new trading pairs to their users. `xud` encompasses the following components:
 
 * Integration with [lnd](https://github.com/lightningnetwork/lnd) and [raiden](https://github.com/raiden-network/raiden) nodes.
 * Decentralized order book to locally aggregate orders from the known network.
 * Matching engine to match new local orders with existing local and remote orders and initiate atomic swaps with remote peers.
 * Peer-to-peer networking with and discovery of other nodes.
-* gRPC API with web proxy to serve other applications, also accessible via the command-line interface `xucli`.
+* [gRPC API](https://grpc.io/) with web proxy to serve other applications, also accessible via the command-line interface `xucli`.
+
+## Documentation
 
-## Install
+The daemon has been designed to be as developer friendly as possible in order to facilitate application development on top of `xud`.
+* [docs.exchangeunion.com](https://docs.exchangeunion.com): the starting point and up-to-date collection of guides, links and examples
+* [api.exchangeunion.com](https://api.exchangeunion.com): the direct link to the automatically generated gRPC API documentation
```

Comment:
I think we can drop "the direct link to" in this line and below and just say "The automatically generated gRPC API documentation."

Reply:
Fixed